randovania/games/prime2/json_data/header.json

```diff
@@ -948,8 +948,8 @@
                 "extra": {}
             },
             "NoSuits": {
-                "long_name": "Suitless Ingclaw/Ingstorm",
-                "description": "If this trick is enabled, Samus may be expected to traverse through Ingclaw and Ingstorm without the Dark Suit or the Light Suit. Instead of the appropriate suits, Energy Tanks may be provided by the logic as a means to pass these obstacles.",
+                "long_name": "Suitless Dark Aether",
+                "description": "If this trick is enabled, Samus may be expected to traverse through the Dark Aether without a suit or with low energy. Higher levels of this trick expects going through Ingclaw or Ingstorm without Dark Suit or Light Suit, respectively. Instead of the appropriate suits, Energy Tanks may be provided by the logic as a means to pass these obstacles.",
```

Comment:
> Samus may be expected to traverse through the Dark Aether without a suit or with low energy.

Not quite a fan of this wording, as it could be understood as "dark aether is only logical if i have a suit" which is not the case. Maybe something more like "Samus may be expected to traverse through parts of the Dark Aether which have barely or no heal spheres, without a suit or with low energy"?

Reply:
Already changed to DCR's suggested description, which imo is a good description.

Reply:
DCRs suggestion doesnt really fix my issue with the "through dark aether without a suit"-part

Reply:
it stating: ...without a suit **or with low energy** ...

Reply:
Ah whoops, my brain missed that. My bad.